src/main/java/org/jabref/gui/texparser/FileNodeViewModel.java

```diff
@@ -49,10 +50,10 @@ public String getDisplayText() {
 
     @Override
     public String toString() {
-        return new StringJoiner(", ", FileNodeViewModel.class.getSimpleName() + "[", "]")
-                .add("path=" + path)
-                .add("fileCount=" + fileCount)
-                .add("children=" + children)
+        return new StringJoiner(", ", getClass().getSimpleName() + '[', "]")
```

Comment:
I like the standard auto-generated Intellj code a bit more...

Reply:
I have been testing different ways and, to be honest, I don't like any of them... :laughing:

Just thought that a `StringJoiner` would be more efficient than many `String` concatenations. Because you talk about this, don't you?

```
    @Override
    public String toString() {
        return "Citation{" +
                "path=" + path +
                ", line=" + line +
                ", colStart=" + colStart +
                ", colEnd=" + colEnd +
                ", lineText='" + lineText + '\'' +
                '}';
    }
```
I know that it is not a recurring method, but maybe using `String.format()`?

Reply:
StringJoiner might be more efficient but since `toString` is usually only called in failing tests (or while debugging) performance shouldn't be a consideration. In the end it doesn't really matter and I don't have a strong opinion which format to use.